### DISCUSSION TITLE
Conflict with keypress

### DISCUSSION THREAD

**Discussion by myregistration** [alpinejs/alpine#3006] [7. Bugs (archived)]
If I use `@keyup.left.window` and `@keyup.left.shift.window` it will do both.  Is this correct behavior and if so, how should I use a combo of keys that are similar to another?  I could use a different key of course, but wondered how to use similar keys/combos.  Thanks!

**Comment by magnusskare** [comment]
You would probably have to write exceptions in your function and exit early for this, as there are use cases where multiple keys can and should be pressed at once (e.g. games).
